dashboard/app/models/unit_group.rb

```diff
@@ -238,7 +238,7 @@ def update_scripts(new_units, alternate_units = nil)
     new_units_objects.each_with_index do |unit, index|
       unit_group_unit = UnitGroupUnit.find_or_create_by!(unit_group: self, script: unit) do |ugu|
         ugu.position = index + 1
-        unit.update!(published_state: nil, instruction_type: nil, participant_audience: nil, instructor_audience: nil)
+        unit.update!(published_state: nil, instruction_type: nil, participant_audience: nil, instructor_audience: nil, is_course: false, version_year: nil, family_name: nil)
```

Comment:
@bethanyaconnor I'm really interested in your thoughts around adding and removing units. What fields should be set and unset?

Reply:
This makes sense to me. I wonder if we need to check if it has a course version and delete it though? If the unit had a course_version then I'm not sure this on its own would destroy it (though it would probably get destroyed on the next seed).

This probably could be looked at in a separate PR.

Reply:
Updated!

Reply:
Awesome thank you!